.github/actions/auto-resolve-done-jira-issue/Update-JiraIssue.ps1

```diff
@@ -0,0 +1,36 @@
+param($Repository, $PullRequestNumber, $IsDone, $IsResolve)
```

Comment:
This is like this because [where the script is called from](https://github.com/Lombiq/GitHub-Actions/blob/4d01a30d478f7021dbc80fe43eb9ed6fb715166e/.github/actions/auto-resolve-done-jira-issue/action.yml#L39) we have two boolean values coming from other steps. So, there needs to be some logic that transforms these into the strings Done and Resolve and does input validation. Now, that could be where this script is called, but then we're just moving some PS code from this analyzed and easy to edit script to the unanalyzed and harder to edit YML.

Since the script is internal to this action, its API doesn't need to be user-friendly to uninitiated users. So, I figure this is better to keep in the script.

Reply:
I see OK!